src/ivectorbin/agglomerative-cluster.cc

```diff
@@ -46,11 +46,11 @@ int main(int argc, char *argv[]) {
       "   ark,t:labels.txt\n";
 
     ParseOptions po(usage);
-    std::string reco2num_rspecifier;
+    std::string reco2num_spk_rspecifier;
     BaseFloat threshold = 0.0;
     bool read_costs = false;
 
-    po.Register("reco2num-rspecifier", &reco2num_rspecifier,
+    po.Register("reco2num_spk-rspecifier", &reco2num_spk_rspecifier,
```

Comment:
I think our convention is to use "-" instead of "_", so `reco2num-spk-rspecifier`, not `reco2num_spk-rspecifier`